src/PostComments.jsx

```diff
@@ -0,0 +1,33 @@
+import { useEffect, useState } from "react";
+import { postComment } from "./Api";
+
+const PostComments = ({ reviewId, setNewComment }) => {
+  const [comment, setComment] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+
```

Comment:
Few bugs associated with this component, if an error occurs we get a screen showing the axios error.

1 key example is if we try to post an empty comment, we just an error and "Posting Comment" showing after the error. I think you need a catch block to deal with errors is 1 thing.

You also want to have some way of preventing this kind of request happening (400 errors).